compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/KeyboardOverlapHeight.uikit.kt

```diff
@@ -17,11 +17,19 @@
 package androidx.compose.ui.uikit
 
 import androidx.compose.runtime.InternalComposeApi
-import androidx.compose.runtime.staticCompositionLocalOf
+import androidx.compose.runtime.compositionLocalOf
 
 
 /**
  * Composition local for height that is overlapped with keyboard over Compose view.
  */
 @InternalComposeApi
-val LocalKeyboardOverlapHeight = staticCompositionLocalOf { 0f }
+val LocalKeyboardOverlapHeight = compositionLocalOf { 0f }
+
+/**
+ * Composition local for Selection Handlers vertical offset.
+ * Applied when [OnFocusBehavior.FocusableAboveKeyboard] used and
+ * [ComposeUIViewControllerConfiguration.platformLayers] are enabled.
+ */
+@InternalComposeApi
+val LocalTextSelectionHandlersOffset = compositionLocalOf { 0f }
```

Comment:
Let's combine everything required to software keyboard in a single class/composition local instead of a few public floats

Reply:
Done